daemon/cmd/daemon_main.go

```diff
@@ -1288,6 +1288,14 @@ func runDaemon() {
 	bootstrapStats.k8sInit.End(true)
 	restoreComplete := d.initRestore(restoredEndpoints)
 
+	if !d.endpointManager.HostEndpointExists() {
+		log.Info("Creating host endpoint")
```

Comment:
What is the purpose of this log message?

Reply:
Simply an indication of where we are in the main initialization (and whether we needed to create host endpoint). We have these sorts of info. messages at [several places in `runDaemon()`](https://github.com/cilium/cilium/blob/f13888d203cc58d9c27346d62d5f085ae936417d/daemon/cmd/daemon_main.go#L1268) and during [the initialization of the health endpoint](https://github.com/cilium/cilium/blob/26308b6304ec9bd1326ac1cdb715ab51cb5b198a/daemon/cmd/health.go#L42) as well.

Happy to remove if we think that one is superfluous.